runtime/android/core_internal/src/org/xwalk/core/internal/XWalkMediaPlayerResourceLoadingFilter.java

```diff
@@ -25,12 +25,12 @@
     @Override
     public boolean shouldOverrideResourceLoading(MediaPlayer mediaPlayer,
             Context context, Uri uri) {
-        if (uri.getScheme().equals(AndroidProtocolHandler.APP_SCHEME)) {
+        String scheme = uri.getScheme();
+        if (scheme == null) return false;
```

Comment:
This is kind of danger to identify schema null is for the cache uri. For eg: aaa, /root all are negative. Please check the cache case explicitly.